pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -250,13 +250,13 @@ cdef class HashTable:
 
 {{py:
 
-# name, dtype, null_condition, float_group
-dtypes = [('Float64', 'float64', 'val != val', True),
-          ('UInt64', 'uint64', 'False', False),
-          ('Int64', 'int64', 'val == iNaT', False)]
+# name, dtype, float_group, default_na_value
```

Comment:
when is float_group used? seems superfluous?

Reply:
Seems to be used in `unique`.